tests/modules/images/fetchpriority/fetchpriority-tests.php

```diff
@@ -38,6 +38,10 @@ public static function tear_down_after_class() {
 	}
 
 	public function test_fetchpriority_img_tag_add_attr_based_on_context_and_loading_lazy() {
+		if ( version_compare( get_bloginfo( 'version' ), '6.3', '>=' ) ) {
```

Comment:
These would be better placed in a `setUp()` method in this class so all tests are skipped if not applicable. See: https://docs.phpunit.de/en/9.6/incomplete-and-skipped-tests.html#skipping-tests

I also am curious if we can make use of the logic we use to determine if the module can be enabled in `can-load.php` rather than creating a specific version check?